docs/blogs/posts/2024/2024-01-parakeet.md

```diff
@@ -0,0 +1,102 @@
+
+* [Parakeet RNNT 1.1B](https://huggingface.co/nvidia/parakeet-rnnt-1.1b) - Best recognition accuracy, modest inference speed. Best used when the most accurate transcriptions are necessary.
+* [Parakeet CTC 1.1B](https://huggingface.co/nvidia/parakeet-ctc-1.1b) - Fast inference, strong recognition accuracy. A great middle ground between accuracy and speed of inference.
+* [Parakeet RNNT 0.6B](https://huggingface.co/nvidia/parakeet-rnnt-0.6b) - Strong recognition accuracy and fast inference. Useful for large-scale inference on limited resources.
+* [Parakeet CTC 0.6B](https://huggingface.co/nvidia/parakeet-ctc-0.6b) - Fastest speed, modest recognition accuracy. Useful when transcription speed is the most important.
+
+<!-- more -->
+
+Parakeet models exhibit resilience against non-speech segments, including music and silence, effectively preventing the generation of hallucinated transcripts.
+
+Built using the [NVIDIA NeMo toolkit](https://github.com/NVIDIA/NeMo), Parakeet prioritizes user-friendliness and flexibility. With pre-trained checkpoints readily available, integrating the model into your projects is a breeze. Whether looking for immediate inference capabilities or fine-tuning for specific tasks, NeMo provides a robust and intuitive framework to leverage the model's full potential.
+
+Key benefits of Parakeet models:
+
+* **State-of-the-art accuracy:** Superior WER performance across diverse audio sources and domains with strong robustness to non-speech segments.
+* **Different model sizes:** Two models with 0.6B and 1.1B parameters for robust comprehension of complex speech patterns.
+* **Open-source and extensibility:** Built on NVIDIA NeMo, allowing for seamless integration and customization.
+* **Pre-trained checkpoints:** Ready-to-use models for inference or fine-tuning.
+* **Permissive license:** Released under CC-BY-4.0 license, model checkpoints can be used in any commercial application.
+
+Parakeet is a major step forward in the evolution of conversational AI. Its exceptional accuracy, coupled with the flexibility and ease of use offered by NeMo, empowers developers to create more natural and intuitive voice-powered applications. The possibilities are endless, from enhancing the accuracy of virtual assistants to enabling seamless real-time communication.
+
+The Parakeet family of models achieves state-of-the-art numbers on the [HuggingFace Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard). Users can try out the [parakeet-rnnt-1.1b](https://huggingface.co/nvidia/parakeet-rnnt-1.1b) firsthand at the [Gradio demo](https://huggingface.co/spaces/nvidia/parakeet-rnnt-1.1b). To access the model locally and explore the toolkit, visit the [NVIDIA NeMo Github page](https://github.com/NVIDIA/NeMo). 
```

Comment:
Maybe we can add a screenshot here? IMO people appreciate screenshots quite a lot.